### PR TITLE
[15.0][IMP] github_connector: Improve paths

### PR DESCRIPTION
**Pull request by victoralmau**
Improve paths

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT40304

### CODE REVIEW

File: github_connector/demo/github_analysis_rule_demo.xml [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
 -->
 <odoo>
     <record id="github_analysis_rule_python" model="github.analysis.rule">
         <field name="name">Python files</field>
         <field name="group_id" ref="github_analysis_rule_group_1_demo" />
-        <field name="paths">*.py</field>
+        <field name="paths">
+            *.py
+            !/setup
+            !.*
```
Comment (legalsylvain) [inline github_connector/demo/github_analysis_rule_demo.xml:13]: what this pattern is excluding ?

File: github_connector/demo/github_analysis_rule_demo.xml [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
 License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 <odoo>
     <record id="github_analysis_rule_python" model="github.analysis.rule">
         <field name="name">Python files</field>
         <field name="group_id" ref="github_analysis_rule_group_1_demo" />
-        <field name="paths">*.py</field>
+        <field name="paths">
+            *.py
+            !/setup
```
Comment (legalsylvain) [inline github_connector/demo/github_analysis_rule_demo.xml:12]: nice catch, thanks !

File: github_connector/demo/github_analysis_rule_demo.xml (hunk 1 above)
Comment (victoralmau) [reply github_connector/demo/github_analysis_rule_demo.xml:13]: This is intended to exclude unnecessary files for analysis (.pre-commit-config.yaml, .travis.yaml, etc).

Comment (legalsylvain) [reply github_connector/demo/github_analysis_rule_demo.xml:13]: OK ! Thanks !

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**